### PR TITLE
Version 1 notebook for comparing CEO labels

### PR DESCRIPTION
**Pull request by hannah-rae**
This notebook currently does the bare minimum functions to compare two CEO labeling projects and get a list of points for which labelers disagree, which can be used to review points as a team and then update labels on CEO with the consensus label.

### CODE REVIEW

**Comment by hannah-rae** [comment]
@ivanzvonkov For this first version I used hardcoded local paths to CEO projects to get the core functionality out there. I was thinking we would probably want to use dvc to load the raw_labels and use those paths, however the downsides of that could be:

1. Requires the user to have created an openmapflow project for the labels they are comparing (which maybe they already have in most/all cases)
2. Requires to user to have added the CEO labels to dvc previously, which may be a tedious step to have to do before comparing.

How do you think we should handle this?

File: openmapflow/notebooks/compare_ceo_labels.ipynb [hunk 1]
```diff
@@ -0,0 +1,2563 @@
+{
```
Comment (ivanzvonkov) [inline openmapflow/notebooks/compare_ceo_labels.ipynb:1]: <p>What about something like:</p><pre class="ql-syntax" spellcheck="false">ceo_set1_path = # Your first csv label file, e.g., ceo-Hawaii-Jan-Dec-2020-(Set-1)-sample-data-2022-08-16.csv
</pre><p>To make it more clear that this is your own path</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/openmapflow/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (ivanzvonkov) [inline openmapflow/notebooks/compare_ceo_labels.ipynb:1]: <p>minor nit: I think some of the cells in this notebook could be squashed into one cell and like this and the above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/openmapflow/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Comment (ivanzvonkov) [inline openmapflow/notebooks/compare_ceo_labels.ipynb:1]: <p>Can the notebook outputs be cleared?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/openmapflow/pull/80/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Comment by ivanzvonkov** [comment]
Nice label emoji 😄 

I think how it's handled depends on the primary use case.
Since this step is going to be done before the data is even added to the project, I think it makes sense to keep it completely independent and have users manually enter paths to their file.

**Comment by hannah-rae** [comment]
@ivanzvonkov made the changes you suggested and also some minor formatting updates to make flake8 happy - let me know if you see anything else!

**Review by ivanzvonkov — APPROVED** [review]

**Pull request merged**